db/migrate/20171109155919_add_employer_information_to_members.rb

```diff
@@ -0,0 +1,14 @@
+class AddEmployerInformationToMembers < ActiveRecord::Migration[5.1]
+  def up
+    add_column :members, :employed_monthly_employer, :string, array: true
```

Comment:
monthly employer?

Reply:
maybe it should be `employed_employer_name` ?

Reply:
There's already a field called `employed_employer_name` unfortunately (that's for SNAP purposes) and it's a single string type, not an array of strings. So it muddies this up a little bit.

All this smells sufficiently like something that should end up being extracted to its own model. `Job`? `MemberJob`?

Also - I called it `employed_monthly_employer` so that it stays as close to the income amount column as possible. It's not the perfect name but it's similar enough to the income column where you can see they're related.

Reply:
@jessieay Made a ticket! https://www.pivotaltracker.com/story/show/152765054